genome_browser/_genome_diagram.py

```diff
@@ -10,11 +9,11 @@
 
 
 class GenomeDiagram(object):
-    def __init__(self, targets, name=None):
-        self.targets = [targets] if not isinstance(targets, (list, tuple)) else targets
+    def __init__(self, name=""):
```

Comment:
No more need for sentinels like `if not self.name` / `if self.name is not None`.
I've yet to see a situation where this causes any problems.